README.md

```diff
@@ -31,7 +31,11 @@ Please see our [contribution guidelines](https://github.com/SkriptLang/Skript/bl
 before reporting issues.
 
 ## Help Us Test
-We have an [official Discord community](https://discord.gg/ZPsZAg6ygu) for testing Skript's new features and releases.
+Wanting to help test Skript's new features and releases?
+You can head on over to our [Official Testing Discord](https://discord.gg/ZPsZAg6yg), and whenever we start testing new features/releases you will be the first to know.
```

Comment:
Link is broken, you're missing a `u` at the end `ZPsZAg6ygu`

Reply:
Thanks for catching that